addons/sale_pdf_quote_builder/models/sale_pdf_form_field.py

```diff
@@ -35,10 +35,12 @@ class SalePdfFormField(models.Model):
              "Leave empty to be able to customized it in the quotation form."
     )
     product_document_ids = fields.Many2many(
-        string="Product Documents", comodel_name='product.document'
+        string="Product Documents", comodel_name='product.document',
+        groups='base.group_user',
```

Comment:
why? non-internal users are not expected to access this model (unless they're in sudo) anyway.
I'd say that there must be a missing sudo somewhere if this group has to be added :thinking: